### PR TITLE
Add tree for the iBridges CLI

### PR DESCRIPTION
**Pull request by qubixes**
This PR adds a new feature to the CLI that makes it possible to print the collection/subcollection tree.

Currently there is an issue with the tree being slow + subcollections showing up twice. This should be fixed after #158 is merged. I suggest we merge that PR first before merging this one.

### CODE REVIEW

File: ibridges/__main__.py [hunk 1]
```diff
@@ -318,3 +320,85 @@ def ibridges_sync():
...
                   dry_run=args.dry_run,
         )
+
+# prefix components:
+space =  '    '
+branch = '│   '
+skip = "..."
+# pointers:
+tee =    '├── '
+last =   '└── '
```
Comment (trel) [inline ibridges/__main__.py:330]: consider/remember an ascii-only presentation.

Comment (qubixes) [reply ibridges/__main__.py]: Yes, that sounds like a good idea. I have added an optional `--ascii` option for those that need it. Thanks!

Comment (trel) [reply ibridges/__main__.py]: i see!  looks great.

Comment (qubixes) [reply ibridges/__main__.py]: Thanks!

**Pull request merged**